### PR TITLE
Adding filter for msiexec repair option

### PR DESCRIPTION
**Pull request by redsand**
see for details, not found in msiexec /h : https://www.robvanderwoude.com/msiexec.php

### CODE REVIEW

File: rules/windows/process_creation/sysmon_always_install_elevated_windows_installer.yml [hunk 1]
```diff
@@ -29,7 +29,9 @@ detection:
     image_2:
         Image|endswith:
             - '\msiexec.exe'
-    condition: (image_1 and user) or (image_2 and user and integrity_level)
+    image_2_flush:
+        Command|endswith: '\system32\msiexec.exe /V' # ignore "repair option"
```
Comment (frack113) [inline rules/windows/process_creation/sysmon_always_install_elevated_windows_installer.yml:33]: `CommandLine` or `ParentCommandLine` ?

**Comment by redsand** [comment]
Command for sure, otherwise it would also be ParentImage also

**Comment by frack113** [comment]
sorry I wasn't clear enough.
In the rule we use the original(1) filed name so for process_creation category it is  `CommandLine`.
When convert use mapping to get your `Command`.

(1) except space that are replace by '_'

**Comment by redsand** [comment]
yes! so sorry! gosh... i didnt translate back in my head. thank you!

**Pull request merged**